### PR TITLE
Admin add user

### PR DESCRIPTION
**Pull request by nhat15**
## Related Tickets
- [#53104](https://edu-redmine.sun-asterisk.vn/issues/53104)

## WHAT (optional)
- CRUD for admin user page

## HOW
- I edit js file, inject not_vary_normal items in calculate function.

## WHY (optional)
- Because in previous version - number just depends on `normal` items. But in new version, we have `state` and `confirm_state` depends on both `normal` + `not_normal` items.

## Evidence (Screenshot or Video)
Screenshot from 2022-09-08 10-32-35
Screenshot from 2022-09-08 10-32-42

### CODE REVIEW

**Comment by nhat15** [comment]
ready

File: app/controllers/admin/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+class Admin::UsersController < Admin::AdminController
+  before_action :find_user, only: %i(show)
+  before_action :load_payment, only: %i(show)
```
Comment (ghost) [inline app/controllers/admin/users_controller.rb]: gộp lại 1 dòng em ơi

File: app/views/admin/payments/_show.html.erb [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+                                            as: :ticket %>
+            </tbody>
+          </table>
+        </div>
+        <%= render partial: "total", locals: {payment: payment} %>
+      </div>
+      <div class="modal-footer"></div>
+    </div>
+  </div>
+</div>
```
Comment (ghost) [inline app/views/admin/payments/_show.html.erb]: thiếu endline

**Review by ghost — COMMENTED** [review]
self review lại em nhé

**Comment by nhat15** [comment]
ready

**Review by ghost — COMMENTED** [review]
merged

**Pull request merged**